### ISSUE TITLE
An unhandled exception occurred: Could not find module "@angular-devkit/build-angular"

### ISSUE THREAD

**Issue by Dave839383** [srwebdev5515/jl-test#2]
we can't run or build your code 

An unhandled exception occurred: Could not find module "@angular-devkit/build-angular"

Can you please fix

**Comment by srwebdev5515** [comment]
@Dave839383, will you do `npm install` again?
It worked fine on my end, and I can't duplicate the issue.

**Comment by Dave839383** [comment]
yep sorry, we got it running but it's hard to find answers to all the questions.  I can see a list of users which looks like it's an attempt at question 4 but can't see answers to the first three questions.
